tests/regression

```diff
@@ -1,10 +1,12 @@
 #!/usr/bin/env bash
+# shellcheck disable=SC2046,SC2086,SC2002
+
```

Comment:
Any particular reason to disable these?

https://github.com/koalaman/shellcheck/wiki/SC2046
https://github.com/koalaman/shellcheck/wiki/SC2086
https://github.com/koalaman/shellcheck/wiki/SC2002

Reply:
Yes, it would require very many edits to correct.